### DISCUSSION TITLE
without disturbing plotting, can I update config file?

### DISCUSSION THREAD

**Discussion by syadnom** [ericaltendorf/plotman#401] [Q&A] [answered]
I'm running in interactive mode in a screen session.  If I add more drives and update the config file, can I just exit plotman and start again without adverse affects?  I have a bunch of staggered plots running and I really don't want to kill them off to test the theory out.

**Comment by altendky** [comment, marked as answer]
The plot processes are independent of plotman and won't be stopped by shutting down plotman.  You can just press `q` from interactive and then rerun plotman to reload the configuration.

**Comment by cojarbi** [comment]
What does p inside Plotman interactive?

On another note, can i suspend, updated Chia and the resume?

**Reply by syadnom** [reply]
from plotman's side, yes, because all plotman does is read logs and fire off plots in the background.  But for chia, if you have any active plots, that might break your current plots because you'll overwrite the executable.  Dangerous...

**Reply by Viking8** [reply]
What about stopping all current activity so I can upgrade to a better CPU? Would chia and plotman allow me to resume from where I stopped plotting? I currently have 4 going in the first phase and each are showing over 100GB in tmp. Wouldn't want to waste all of that downloaded info.

**Reply by syadnom** [reply]
It's a chia thing.   Linux once had patches for checkpoint/restart of a process but I don't know the status on that.  So either chia needs to implement restarting, or you have to find some hack in the kernel to make it happen.  no small feat.

**Reply by Viking8** [reply]
I've been around linux since 1998, but not as a full time user. That sounds way beyond my abilities. Although, I am really good at following VERY detailed instructions! 😉
